scipy/ndimage/interpolation.py

```diff
@@ -62,12 +63,24 @@ def spline_filter1d(input, order=3, axis=-1, output=numpy.float64):
     output : ndarray or dtype, optional
         The array in which to place the output, or the dtype of the returned
         array. Default is `numpy.float64`.
+    %(mode)s
 
     Returns
     -------
     spline_filter1d : ndarray
         The filtered input.
 
+    Notes
+    -----
+    All functions in `ndimage.interpolation` internally use a b-spline
+    representation of the input image. Whenever using b-splines of
+    `order > 1`, the input image values have to be converted to b-spline
+    coefficients first, which is done by applying this one-dimensional
+    spline filter sequentially along all axes of the input. All of the
+    functions that require such an input will automatically filter their
```

Comment:
"need the b-spline coefficients" or some such.

Reply:
PTAL.